tap-gui/plugins/runtime-resource-visibility.md

```diff
@@ -66,15 +65,82 @@ applicable for the resource type.
 To view the list of your running resources:
 
 1. Select the Component from the Catalog index page.
-1. Select the Workloads tab.
+2. Select the Runtime Resources tab.
 
-    ![Workload index table](./images/workload-visibility-workloads.png)
+![Runtime resources index table collapsed](images/runtime-resources-index.png)
 
-## Knative service details page
+### Seeing details for a specific resource
+
+The index table will show Deployments, Pods, ReplicaSets and Services that matches with the label indicated in the component's definition; you will see a master-detail structure between all the different objects using the owners and dependents relationships stablished between them; those resources without an owner will be listed in the table as independent elements.
+
+> For additional information about owners and dependants please visit [the official documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/)
+
+Following, you can see an example of an expanded view for the index table.
+
+![Runtime resources index table collapsed](images/runtime-resources-expanded.png)
+
+## Detail pages
+
+Currently, we provide pages for many resources to allows users to see the most relevant characteristics of each one, including direct links to other resources.
+These are the common sections included in all pages:
+
+### Overview section
+
+It is the first card in all details pages, most of the information included in it came from the `metadata` attribute in each object. Some attributes displayed here are:
+
+1. **.YAML** button: Show the current object's definition in yaml; you can copy the full content using the icon in the top-right corner.
+2. Name
+3. Namespace
+4. Age / Creation date
+5. Cluster*
+6. URL**
+
+**Cluster: the value displayed corresponds to the name used in the cluster's configuration.*
+
+** _URL: Available for knative services and K8s services_
+
+E.g:
+
+![Overview section](images/runtime-resources-overview.png)
+
+### Status section
+
+This section displays the all the conditions included in the resource's attribute `status.conditions`; not all resources has conditions, they could be different between each one.
+
+For a better understanding about status please visit [Concepts - Object Spec and Status](https://kubernetes.io/docs/concepts/_print/#object-spec-and-status).
+
+![Status section](images/runtime-resources-status.png)
+
+### Ownership section
+
+Depending on the resource that you are currently viewing, this section presents all the resources specified in the `metadata.ownerReferences`.
+
+> For additional information about owners and dependants please visit [the official documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/)
```

Comment:
(let's use "dependents" instead of "dependants" to keep it consistent with the k8s documentation)